projects/element-ng/tabs-next/si-tabset-next.component.ts

```diff
@@ -100,16 +100,26 @@ export class SiTabsetNextComponent implements AfterViewInit {
   }
 
   /** @internal */
-  getNextIndexToFocus(currentIndex: number): number {
+  removedTabByUser(index: number, active: boolean): void {
+    // The tab was already removed from the tabPanels list when this function is called.
+    // We need to:
+    // - focus another tab if the closed one was focused
+    // - activate another tab if the closed one was active
+    // If the closed tab was not focussed, there is no need to restore the focus as it could only be closed by mouse.
     for (let i = 0; i < this.tabPanels().length; i++) {
       // Get the actual index using modulo to wrap around
-      const checkIndex = (currentIndex + i) % this.tabPanels().length;
-
-      if (!this.tabPanels()[checkIndex].disabledTab()) {
-        return this.tabPanels()[checkIndex].index();
+      const checkIndex = (index + i) % this.tabPanels().length;
```

Comment:
@spike-rabbit This has nothing to do with this PR but the logic here is actually a bit shitty/basic IMHO. In Material or e.g. in Browsers usually the previous tab (in the sense of the tab to the left of the closing one) is being activated. Maybe this would be the chance to adjust this behavior in another PR.

@siemens/siemens-element Anyone up to do this quick change?